client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -623,27 +678,29 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
           />
         </div>
       </ExpandableSection>
-      <ActionGroup>
-        <Button
-          type="submit"
-          id="submit"
-          aria-label="submit"
-          variant={ButtonVariant.primary}
-          isDisabled={!isValid || isSubmitting || isValidating || !isDirty}
-        >
-          {!application ? t("actions.create") : t("actions.save")}
-        </Button>
-        <Button
-          type="button"
-          id="cancel"
-          aria-label="cancel"
-          variant={ButtonVariant.link}
-          isDisabled={isSubmitting || isValidating}
-          onClick={onClose}
-        >
-          {t("actions.cancel")}
-        </Button>
-      </ActionGroup>
+      {showFormButtons && (
```

Comment:
Can we just remove this code since the buttons live in the modal now?

Reply:
yes. I was not sure what's the plan for this form so I've kept the refactoring minimal.

Reply:
If you want to remove the buttons that is fine by me. I appreciate you making the effort to minimize changes.

Reply:
Removed the buttons as suggested.